client/launcher.py

```diff
@@ -467,7 +467,7 @@ def process_request(service_list, cmd, subcmd, is_json, args, auth=None):
             raise ValueError("ERROR: service '%s' not defined" % args.service)
 
         if args.gpus is None:
-            if "trans" in args.docker_command:
+            if "trans" in args.docker_command or "score" in args.docker_command:
```

Comment:
It would be better like that ?
```suggestion
            if "train" in args.docker_command:
                args.gpus = 1
            else:
                args.gpus = 0
```